src/Setfield.jl

```diff
@@ -8,4 +8,12 @@ include("sugar.jl")
 include("functionlenses.jl")
 include("settable.jl")
 include("experimental.jl")
+
+for n in names(Setfield, all=true)
+    T = getproperty(Setfield, n)
+    if T isa Type && T <: Lens && (T === ComposedLens || has_atlens_support(T))
+        @eval Base.show(io::IO, l::$T) = _show(io, nothing, l)
+    end
+end
```

Comment:
Can you add a comment that mentions the stack overflow and this PR?

Reply:
See bd1594aedf9a6ef68779c6fb537cf4887cafc61b